engine/src/main/java/org/terasology/engine/subsystem/headless/mode/StateHeadlessSetup.java

```diff
@@ -103,6 +105,9 @@ public void init(GameEngine gameEngine) {
 
         componentSystemManager.initialise();
 
+        // Get the server to start broadcast
+        broadCastServer.startBroadcast();
```

Comment:
This won't work as it isn't initialized yet if you compare to the other PR it uses `@In` to retrieve that object from the game `Context`

Reply:
Ohk.